zproject/dev_settings.py

```diff
@@ -62,6 +62,13 @@
 )
 
 EXTERNAL_URI_SCHEME = "http://"
+
+truthy_strings = ("y", "yes", "t", "true", "on", "1")
```

Comment:
I borrowed this from [distutils' `strtobool`](https://github.com/pypa/distutils/blob/main/distutils/util.py#L340).

Feedback is very welcome on how to organize this bit of code.

I couldn't easily find a file that would be a good spot to place this, and didn't know if I should introduce a new file/library in the codebase for this change.

Reply:
I would prefer not to accept more than one value. But thinking about this more, we probably want this to be implemented  as a command-line option to `run-dev`, say `--behind-https-proxy`, and have that program be responsible for configuring the development environment (either via environment or otherwise) here.